III.tdd-katas/calculator-tdd/add.test.js

```diff
@@ -0,0 +1,40 @@
+const add = require("./add");
+describe("Step 2: handle an unknown amount of numbers", () => {
+    test("should return 0 for empty string passed to add", () => {
+        expect(add("")).toBe(0);
+    });
+    test('should return 1 for "1" string passed to add', () => {
+        expect(add("1")).toBe(1);
+    });
+    test('should return 3 for "1,2" string passed to add', () => {
+        expect(add("1,2")).toBe(3);
+    });
+});
+describe("Step 2: handle an unknown amount of numbers", () => {
+    test("should return 0 for empty string passed to add", () => {
+        expect(add("1,2,4fi5faswwe6,lgsf8")).toBe(26);
```

Comment:
I wouldn't expect that; if the user's input contains things that *aren't* numbers separated by the delimiter, they should either be ignored or an error thrown (I'd prefer the latter). And if you are going to sanitise the input I'd expect the result to be 467, because if you remove the characters that aren't numbers or the delimiter you get `'1,2456,8'`.